pkg/term/term_linux_test.go

```diff
@@ -3,29 +3,20 @@
 package term
 
 import (
-	"flag"
 	"io/ioutil"
 	"os"
 	"testing"
 
-	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 )
 
-var rootEnabled bool
-
-func init() {
-	flag.BoolVar(&rootEnabled, "test.root", false, "enable tests that require root")
```

Comment:
@thaJeztah @dnephin I think it's fine like this. We could do what dnephin said in https://github.com/moby/moby/pull/33520/files#r125954995 but honestly if you don't run it as root, you'll see the tests as skipped with a reason. The person reading the test logs should pay attention to skipped tests and take the decision of whether that is acceptable or not. That's how they would discover this behavior.